docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -204,14 +269,23 @@ argument to this attribute or a list of arguments:
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompilationTask
 // ...
 
-val compileKotlin: KotlinCompilationTask<*> by tasks
+kotlin {
+    compilerOptions {
+        // Single experimental argument
+        freeCompilerArgs.add("-Xexport-kdoc")
+
+        // Single additional argument
+        freeCompilerArgs.add("-Xno-param-assertions")
```

Comment:
I would prefer to use here more specialized DSL than `freeCompilerArgs` - for example `apiVersion` and `jvmTarget`.

Kotlin:
```
compilerOptions {
    apiVersion.set(KotlinVersion.KOTLIN_2_0)
    jvmTarget.set(JvmTarget.JVM_1_8)
}
```

Groovy:
```
compilerOptions {
    apiVersion = KotlinVersion.KOTLIN_2_0
    jvmTarget = JvmTarget.JVM_1_8
}
```

Reply:
@Tapchicoma I guess in kts files you can also use assign operator on lazy api at least in latest Gradle versions, isn't it?
```kotlin
compilerOptions { 
    apiVersion = KotlinVersion.KOTLIN_2_0
    jvmTarget = JvmTarget.JVM_1_8
}
```

Reply:
Yes, it is possible to use, though only starting Gradle 8.2+ versions. As we are still supporting Gradle 7.6.3 we need to provide an example without assign operator.

Reply:
I see, thank you for explaining your objection